CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.2.1
```

Comment:
If there is an API change (such as a new export), it should be `0.5.3.0`. The fourth digit is just for packaging changes, see https://pvp.haskell.org for details.

Reply:
Done